ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_manipulation.py

```diff
@@ -637,3 +639,67 @@ def test_paddle_take_along_axis(
         indices=indices,
         axis=axis,
     )
+
+
+@st.composite
+def put_along_axis_helper(draw):
+    input_dtype, x, axis, shape = draw(
+        helpers.dtype_values_axis(
+            available_dtypes=helpers.get_dtypes("valid"),
+            min_num_dims=2,
+            min_dim_size=2,
+            valid_axis=True,
+            force_int_axis=True,
+            ret_shape=True,
+            min_axis=0,
+            max_axis=1,
```

Comment:
Why did you choose to set `max_axis=1` here? I don't think we need to limit the test like this.